CHANGELOG.md

```diff
@@ -8,6 +8,17 @@
 
 * None
 
+
+## 2.1.3 (2021-01-21)
+
+##### Enhancements
+
+* Adds support to not offset a KeyboardScrollable view if the viewController is presented in a popover.
```

Comment:
Can you move this under the Master/Enhancements section and remove the `2.1.3 (2021-01-21)` header? One of us will create the versioned header when we do the release.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/61/commits/864739ee53306fae9894e939272c87c531bb54e8